### PR TITLE
[SchemaRegistry] next-pylint update

### PR DESCRIPTION
**Pull request by swathipil**
Fixing next-pylint errors

### CODE REVIEW

**Comment by swathipil** [comment]
tests-weekly run: https://dev.azure.com/azure-sdk/internal/_build?definitionId=3698&_a=summary

File: sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_schema_registry_avro_encoder.py [hunk 1]
```diff
@@ -71,42 +71,38 @@ class AvroEncoder(object):
...
-        try:
-            self._schema_registry_client = kwargs.pop(
-                "client"
-            )  # type: "SchemaRegistryClient"
-        except KeyError as exc:
-            raise TypeError(f"'{exc.args[0]}' is a required keyword.")
+    def __init__(
+        self,
+        client: "SchemaRegistryClient",
+        group_name: Optional[str] = None,
```
Comment (kashifkhan) [inline sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_schema_registry_avro_encoder.py:77]: was this intentional to change these from being kwargs ?

File: sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py [hunk 2]
```diff
@@ -154,10 +155,12 @@ def get_schema(self, *args: str, **kwargs: Any) -> Schema:
         2) To get a specific version of a schema within the specified schema group, pass in the required
         keyword arguments `group_name`, `name`, and `version` and any optional keyword arguments.
 
-        :param str schema_id: References specific schema in registry namespace.
```
Comment (l0lawrence) [inline sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py:157]: :param args:

**Review by l0lawrence — COMMENTED** [review]
small comments, thanks!

File: sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_schema_registry_avro_encoder.py (hunk 1 above)
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry-avroencoder/azure/schemaregistry/encoder/avroencoder/_schema_registry_avro_encoder.py:77]: yes, changing them to py 3 typing

File: sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py (hunk 2 above)
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py:157]: same as above

**Comment by swathipil** [comment]
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2894626&view=results

**Review by l0lawrence — APPROVED** [review]

**Pull request merged**